components/server/src/billing/billing-mode.spec.db.ts

```diff
@@ -24,8 +24,10 @@ chai.use(deepEqualInAnyOrder);
 const expect = chai.expect;
 
 type StripeSubscription = Pick<Stripe.Subscription, "id"> & { customer: string };
-class UsageServiceMock implements UsageService {
-    constructor(protected readonly subscription?: StripeSubscription) {}
+class UsageServiceMock extends UsageService {
```

Comment:
Out of curiosity, what makes the inheritance here better than implementing an interface?

Reply:
I removed the interface, impl and symbol on the declaration side.

Reply:
I could actually implement the type of that class instead of inheriting it. Would be a tad cleaner, indeed. Will do when I touch this the next time.